Formula/c/cloudiscovery.rb

```diff
@@ -70,11 +70,6 @@ class Cloudiscovery < Formula
     sha256 "90261b206d6defd58fdd5e85f478bf633a2901798906be2ad389150c5c60edbe"
   end
 
-  resource "markupsafe" do
-    url "https://files.pythonhosted.org/packages/bf/10/ff66fea6d1788c458663a84d88787bae15d45daa16f6b3ef33322a51fc7e/MarkupSafe-2.0.1.tar.gz"
-    sha256 "594c67807fb16238b30c44bdf74f36c02cdf22d1c8cda91ef8a0ed8dabf5620a"
-  end
-
```

Comment:
Not sure why this was removed, looks like it's still required

Reply:
dependency is in L22

Reply:
Ah. Unfortunately it looks like something is incompatible with markupsafe 2.1 (https://markupsafe.palletsprojects.com/en/2.1.x/changes/#version-2-1-0)

Reply:
So, I don't think the brew markupsafe formula works here, we need to use the older version from pypi